src/layout.js

```diff
@@ -162,6 +162,11 @@ export function addPadding(nodeArray, padding) {
       i++;
     }
   });
+
+  if (maxY === 0) {
+    maxY = nodeArray.reduce((acc, current) => (acc.in > current.in) ? acc : current).in;
+  }
+
```

Comment:
I think moving the `maxY = Math.max(maxY, node.y + Math.max(node.in, node.out));` from line 161 down couple of lines outside the `if (node.y)` block should fix it? Would need to consider the `out` too anyway.

Reply:
Yes, indeed, that' s enough. I have made the change.